### DISCUSSION TITLE
Gatsby version

### DISCUSSION THREAD

**Discussion by kemalgencay** [gatsbyjs/gatsby#30784] [Help]
Hi,
I've installed globally gatsby version 3.2.0, in a project I've build a while ago the dependencies section sites : "gatsby" : "^2.21.0"
Would that affect my project or not?
TIA
Regards

**Comment by LekoArts** [comment]
Hi!

You shouldn't install `gatsby` globally but only in your project as you might see weird behaviors otherwise. If you want to run commands you can install `gatsby-cli` globally: https://www.gatsbyjs.com/docs/reference/gatsby-cli/

**Reply by kemalgencay** [reply]
Thanks, very helpful indeed
